package.json

```diff
@@ -4,11 +4,13 @@
   "description": "a website for campgrounds",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app.js"
   },
   "author": "Rachamim Dabbah",
   "license": "ISC",
   "dependencies": {
+    "body-parser": "^1.17.2",
```

Comment:
@RayDabbah that's actually a good point. The reason for that is that if you run `npm install foo` without adding the `-save` flag (or `-S` for short) it'll install the foo module into node_modules w/o declaring a dependency on it in package.json. So if you `require('foo')` in the code it'll only work on your machine and everyone else won't automatically have foo installed when they run `npm i`. 👎